### DISCUSSION TITLE
Fishing attempt from a website

### DISCUSSION THREAD

**Discussion by james2doyle** [community/community#144967] [Code Security]
### Select Topic Area

Show & Tell

### Body

![Screen Shot 2024-11-18 at 11 44 30 AM](https://github.com/user-attachments/assets/cc94b42f-9b94-4f07-948e-94273233b2e6)

This URL on the right hand side is pretending to be Github. I tried tweeting at Phind search and Presubmit AI but got no response from them.

**Comment by ghostinhershell** [comment]
Hi @james2doyle, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
